doc/man1/openssl-asn1parse.pod

```diff
@@ -37,16 +37,16 @@ structures. It can also be used to extract data from ASN.1 formatted data.
 
 Print out a usage message.
 
-=item B<-inform> B<DER|PEM>
+=item B<-inform> B<DER>|I<PEM>
```

Comment:
Again here shouldn't it be B<PEM>, or it should be at least consistent.

Reply:
made to be ```I<>```  And fixed the markup in the text description in the lines that follow.